README.md

```diff
@@ -294,19 +295,51 @@ set to `true`, there would be no `accelerator` to contain against. To avoid conf
+| Field Name              | Type                                                                 | Description                                                                                                                                                                                                                                                                                                                                                                                                              |
+|-------------------------|----------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| name                    | string                                                               | **REQUIRED** Name of the input variable defined by the model. If no explicit name is defined by the model, an informative name (e.g.: `"RGB Time Series"`) can be used instead.                                                                                                                                                                                                                                          |
+| description             | string                                                               | Additional details about the input such as describing its purpose or expected source that cannot be represented by other properties.                                                                                                                                                                                                                                                                                     |
+| bands                   | \[string \| [Model Band Object](#model-band-or-variable-object)]     | **REQUIRED**\* <sup>[\[1\]][note-inputs]</sup> The raster band references used to train, fine-tune or perform inference with the model, which may be all or a subset of [Band Object](#bands-and-statistics). If none applies for the input, use an empty array.                                                                                                                                                         |
```

Comment:
I worry it could be confusing to require an empty array instead of not supplying the bands field. Same for the variables field. Can we instead only require one or the other and then if one is supplied we validate that the other field is not present? I don't think a user would ever want to supply both in the same Model Input Object since they describe a similar scope.

Reply:
I think this is fine to remove the Oneof requirement.